README.md

```diff
@@ -159,6 +159,8 @@ RedisSessionHandler solves this problem with a "lock" entry for every session th
 [Session fixation](https://www.owasp.org/index.php/Session_fixation) is the ability to choose your own session ID as an HTTP client. When clients are allowed to choose their
 session IDs, a malicious attacker might be able to trick other clients into using an ID already known to him, then wait for them log in and hijack their session.
 
+**To ensure cookies are replaced, any custom cookie parameters should be registered with session_set_cookie_params() as browsers will only update cookies with matching parameters.**
```

Comment:
Dja think it's necessary to state anything? As far as I understand, now everything works as expected either if you set the custom session cookie params in the php.ini file or with `session_set_cookie_params` (just tested, it really works either way).